src/components/OptionsList/BaseOptionsList.js

```diff
@@ -129,7 +179,11 @@ class BaseOptionsList extends Component {
     renderSectionHeader({section: {title, shouldShow}}) {
         if (title && shouldShow && !this.props.hideSectionHeaders) {
             return (
-                <View>
+
+                // Warning: this optionsListSectionHeader style is brittle – it's computed manually from the dynamic styles in the text node below.
+                // We do this only so that we can reference the height in getItemLayout
+                // 11pt font + 20px padding renders to 54px height
```

Comment:
Can we make it less brittle? What could someone to do accidentally break it?

Reply:
> What could someone to do accidentally break it?

If someone changed the text style of the text component on line 190, switching `styles.textMicroBold` to `textLabel`, increasing the font size from 11pt to 13pt, then the text would appear cut-off, because the wrapper has an explicit height.

We could remove the explicit height from the wrapper, but then making that same change would break `getItemLayout` in a much-less-obvious way than text appearing cut-off.

I'm open to ideas to improve this! It's not perfect, but also works okay.

_Side-note:_ Someday we might hopefully upgrade to a list that doesn't require `getItemLayout` (_theoretically_ shouldn't be necessary with the new RN architecture).

Reply:
> Can we derive this somehow instead of make it explicit?

I can't really think of a way to do this. As stated above, we could just remove the explicit height from the wrapper, but then making that same change would break getItemLayout in a much-less-obvious way than text appearing cut-off, because `getItemLayout` would still be reliant upon that pre-defined 54px height.

It's sort of a chicken-and-the-egg problem ... because `getItemLayout` is used to determine the layout of an item, we can't use the layout of an item to produce the output from `getItemLayout`.

Reply:
> Or maybe be really specific about what someone would need to do if they had to update the text style? I'm not sure from reading this and just feel a general confusion and fear about ever changing it...

Let me see if I can add a more clarifying comment

Reply:
I updated the comment in a way that's more informative, let me know your thoughts